HW2_Zolotenkova/calculator.py

```diff
@@ -0,0 +1,24 @@
+def summa(a, b):
+    answer = float(a) + float(b)
+    return (answer)
+def raznost(a, b):
+    return(a - b)
+def delenie(a, b):
+    result = a / b
+    return result
+def multiply(a,b):
+    answer = a * b
+    return(answer)
+def main():
+    a, znak, b = input().split()

```

Comment:
Лучше давать осознанные названия переменных. Не надо называть переменные и функции транслитерацией. 
```suggestion
    num1, sign, num2 = input().split()

```